src/teamplay.c

```diff
@@ -713,20 +764,38 @@ static void TeamplayReportTaken(gedict_t *client)
 		}
 		else if ((NEED(needFlags, it_health) || NEED(needFlags, it_armor) || NEED(needFlags, it_rl)
 				|| NEED(needFlags, it_lg) || NEED(needFlags, it_rockets)
-				|| NEED(needFlags, it_cells)) && HAVE_POWERUP(client))
+				|| NEED(needFlags, it_cells)))
```

Comment:
brainz hurts, getting near bottom, is there any risk of something not `HAVE_POWERUP(client)` and not `isCTF() && (HAVE_FLAG(client) || HAVE_RUNE(client))` slips through here and produces some weird `"need "` message?